apps/dashboard/lib/env.ts

```diff
@@ -31,6 +31,8 @@ export const env = () =>
       AGENT_TOKEN: z.string(),
 
       GITHUB_KEYS_URI: z.string().optional(),
+
+      UNKEY_ROOT_KEY: z.string(),
```

Comment:
What implication does this have for running unkey either locally or on vercel?

Reply:
It is required for local Dev. Vercel one will need to be added.

Reply:
have you tested this locally?
how did you run it?

Reply:
Used one of My keys and tested locally.

Reply:
ok, please go delete your .env file and run `pnpm local`
my point is: if we merge as is, we broke our local quickstart

Reply:
Did this everything seems to work at the moment with some modifications.

Reply:
what modifications did you make?

Reply:
Made it optional and added a check in the procedure so it uses protected if no rootkey is found.

Reply:
oh nice, that's perfect